roles/redhat-subscriptions/tasks/activation_key.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install katello-ca-consumer-latest
+  when:
+    - ansible_distribution == "RedHat"
+    - ansible_distribution_major_version == "7"
+  yum:
+    name: "{{ repository_url }}/pub/katello-ca-consumer-latest.noarch.rpm"
```

Comment:
Should this variable be name spaced for the role? `redhat_subscriptions_repository_url` ? Or maybe `redhat_subscriptions_satellite_url` ?

Reply:
repository_url is being used in multiple roles, hence using just repository_url.
I believe for `satellite_distribution` and `satellite_version` too as it would be used/required in multiple roles decided not to do that. If any other suggestion around this, let me know.